### PR TITLE
docs issue #1170 xsite expose node port

### LINKED ISSUE #1170: Docs: Set xsite service expose type = NodePort
if you set service.site.local.expose.type=NodePort then you can configure the exposed port number with service.site.local.expose.nodePort=<portnum>

for LoadBalancer or ClusterIP .nodePort field is ignored

### PR DESCRIPTION
**Pull request by oraNod**
fix #1170

### CODE REVIEW

**Review by rigazilla — APPROVED** [review]
lgtm

File: documentation/asciidoc/topics/yaml/xsite_expose_nodeport.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+spec:
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: LON
+        expose:
+          type: NodePort
+          nodePort: 7800
```
Comment (dmvolod) [inline documentation/asciidoc/topics/yaml/xsite_expose_nodeport.yaml:9]: Node port has predefined range 30000-32767 by default
https://kubernetes.io/docs/concepts/services-networking/service/

File: documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc [hunk 2]
```diff
@@ -12,6 +12,15 @@ Configure {ispn_operator} to establish cross-site views with {brandname} cluster
 
 . Create an `Infinispan` CR for each {brandname} cluster.
 . Specify the name of the local site with `spec.service.sites.local.name`.
+. Set the value of the `service.sites.local.expose.type` field to either `NodePort` or `LoadBalancer`.
++
+If you expose the cross-site replication service through a node port, you can also configure a port as follows:
```
Comment (pruivo) [inline documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc:17]:
```suggestion
If you expose the cross-site replication as a `NodePort` service, you can also configure a port as follows:
```

**Review by pruivo — APPROVED** [review]

File: documentation/asciidoc/topics/yaml/xsite_expose_nodeport.yaml (hunk 1 above)
Comment (oraNod) [reply documentation/asciidoc/topics/yaml/xsite_expose_nodeport.yaml]: Thanks @dmvolod I've updated to mention the defaults.

File: documentation/client_cert.md [hunk 3]
```diff
@@ -1,112 +0,0 @@
-# Client Certification
-
-## Server
```
Comment (pruivo) [inline documentation/client_cert.md:3]: @oraNod is this no longer supported?

Comment (oraNod) [reply documentation/client_cert.md:3]: @pruivo sorry, that should actually be removed in https://github.com/infinispan/infinispan-operator/pull/1165 

It's just that I failed to remove Ryan's markdown file when we added client cert authn to documentation/asciidoc.

**Review by pruivo — APPROVED** [review]

**Review by dmvolod — APPROVED** [review]

**Comment by oraNod** [comment]
Self-merging and will backport to 2.1.x.

**Pull request merged**